cmd/main.go

```diff
@@ -49,57 +47,38 @@ func main() {
 	// TODO: Remove this once the config-map based mount approach lands after rebase
 	oldServiceCASrc := fmt.Sprintf("%s/service-ca.crt", builder.SecretCertsMountPath)
 	oldServiceCADst := fmt.Sprintf("%s/service.crt", tlsCertRoot)
-	err = CopyFileIfExists(oldServiceCASrc, oldServiceCADst)
+	err = CopyIfExists(oldServiceCASrc, oldServiceCADst)
 	if err != nil {
 		fmt.Printf("Error setting up service CA cert: %v", err)
 		os.Exit(1)
 	}
 
-	runtimeCASrc := fmt.Sprintf("%s/certs.d", builder.ConfigMapCertsMountPath)
-	err = CopyDirIfExists(runtimeCASrc, runtimeCertRoot)
+	newServiceCASrc := fmt.Sprintf("%s/service-ca.crt", builder.ConfigMapCertsMountPath)
+	newServiceCADst := fmt.Sprintf("%s/openshift-service.crt", tlsCertRoot)
+	err = CopyIfExists(newServiceCASrc, newServiceCADst)
```

Comment:
@bparees I think this restores copying the service-serving CA to the TLS root (the one injected by the controller)